lib/fog/mock/errors.rb

```diff
@@ -0,0 +1,5 @@
+module Fog
+  module Errors
+    class MockNotImplemented < Fog::Errors::Error; end
+  end
+end
```

Comment:
Wouldn't the path of this file be `lib/fog/errors/mock_not_implemented` instead?